diff --git a/lib/module/PropsRegistryGarbageCollector.js b/lib/module/PropsRegistryGarbageCollector.js
--- a/lib/module/PropsRegistryGarbageCollector.js
+++ b/lib/module/PropsRegistryGarbageCollector.js
@@ -2,7 +2,7 @@
 
 import { unprocessColor, unprocessColorsInProps } from "./common/style/processors/colors.js";
 import { ReanimatedModule } from './ReanimatedModule';
-const FLUSH_INTERVAL_MS = 500;
+const FLUSH_INTERVAL_MS = 1100;
 export const PropsRegistryGarbageCollector = {
   viewsMap: new Map(),
   intervalId: null,
diff --git a/src/PropsRegistryGarbageCollector.ts b/src/PropsRegistryGarbageCollector.ts
--- a/src/PropsRegistryGarbageCollector.ts
+++ b/src/PropsRegistryGarbageCollector.ts
@@ -8,7 +8,7 @@
 import type { IAnimatedComponentInternal } from './createAnimatedComponent/commonTypes';
 import { ReanimatedModule } from './ReanimatedModule';
 
-const FLUSH_INTERVAL_MS = 500;
+const FLUSH_INTERVAL_MS = 1100;
 
 export const PropsRegistryGarbageCollector = {
   viewsMap: new Map<number, IAnimatedComponentInternal>(),
